src/librustdoc/core.rs

```diff
@@ -153,14 +156,42 @@ pub fn run_core(search_paths: SearchPaths,
             edition,
             ..config::basic_debugging_options()
         },
+        error_format,
         ..config::basic_options().clone()
     };
 
     let codemap = Lrc::new(codemap::CodeMap::new(sessopts.file_path_mapping()));
-    let diagnostic_handler = errors::Handler::with_tty_emitter(ColorConfig::Auto,
```

Comment:
What is the purpose of these changes?

Reply:
Allow rustdoc output to be checked through UI test suite.